etc/di.xml

```diff
@@ -173,6 +174,18 @@
     </virtualType>
     <!-- Void command -->
 
+    <!-- Capture Fail Void command -->
+    <virtualType name="AffirmGatewayCaptureFailVoidCommand" type="Magento\Payment\Gateway\Command\GatewayCommand">
+        <arguments>
+            <argument name="requestBuilder" xsi:type="object">AffirmGatewayVoidRequest</argument>
+            <argument name="transferFactory" xsi:type="object">AffirmClientGatewayTransferFactory</argument>
+            <argument name="client" xsi:type="object">Astound\Affirm\Gateway\Http\Client\ClientService</argument>
+            <argument name="validator" xsi:type="object">Astound\Affirm\Gateway\Validator\Client\PaymentActionsValidatorCaptureFailVoid</argument>
+            <!--<argument name="handler" xsi:type="object">Astound\Affirm\Gateway\Response\TransactionRefundHandler</argument>-->
```

Comment:
do we need handler?

Reply:
Nope, I copied that from the void section.  we can remove that